### DISCUSSION TITLE
Is it possible to set custom data-attributes on mantine components?

### DISCUSSION THREAD

**Discussion by Haberkamp** [mantinedev/mantine#3039] [Q&A] [answered]
Hello, I wanted to ask if it's possible to set custom data attributes on Mantine components. In my case I want to disable the Dashlane autocompletion on select components, because it is not needed in my specific scenario and it appears to me that it is only possible trough setting a custom data attribute. Thank you!

**Comment by rtivital** [comment, marked as answer]
You can add any data-* and other attributes to the input element.
`<Select data-attribute />`

**Reply by Haberkamp** [reply]
Ahh oke, I think it would be helpful to document that somewhere? Do you mind if I would write some docs for it?

**Reply by rtivital** [reply]
Not really, all components follow this logic, it works the same way as in all other libraries
